src/modules/assets/assets.dto.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  IsNumber,
+  IsOptional,
+  IsString,
+  validateOrReject,
+} from 'class-validator';
+
+export class AssetDto {
+  static async create(data: Partial<AssetDto>) {
+    const dto = new AssetDto();
+    Object.assign(dto, data);
+    await validateOrReject(dto, { whitelist: true });
+    return dto;
+  }
+
+  @IsString()
+  id: string;
+
+  @IsString()
+  owner: string;
+
+  @IsNumber()
+  at: number;
```

Comment:
`at` is the time from chain when the event happened :) it's the number of seconds since `January 01, 1970, 00:00:00` :) so to get JS time we need to multiply this value by `1000` to get milliseconds since `since January 01, 1970, 00:00:00` :)

Reply:
@dwojno Thanks, but why is the property on the `AssetDto`? Is `AssetDto` really an `AssetEventDto`?